### ISSUE TITLE
Uncaught TypeError: Class constructor CompositeLayer cannot be invoked without 'new'

### ISSUE THREAD

**Issue by turban** [teralytics/flowmap.gl#124]
I'm using the interleaved integration between deck.gl and Maplibre GL: https://deck.gl/docs/get-started/using-with-map

It works fine when I test with an ArcLayer: 

```
import { Map } from "maplibre-gl";
import { MapboxLayer } from "@deck.gl/mapbox";
import { ArcLayer } from "@deck.gl/layers";

const map = new Map({
  container: "map",
  style: "https://basemaps.cartocdn.com/gl/positron-gl-style/style.json",
});

map.on("load", () => {
  map.addLayer(
    new MapboxLayer({
      id: "deckgl-arc",
      type: ArcLayer,
      data: [
        {
          source: [-122.3998664, 37.7883697],
          target: [-122.400068, 37.7900503],
        },
      ],
      getSourcePosition: (d) => d.source,
      getTargetPosition: (d) => d.target,
      getSourceColor: [255, 208, 0],
      getTargetColor: [0, 128, 255],
      getWidth: 8,
    })
  );
});
```

When I try the same with FlowMapLayer I get an error: 

Uncaught TypeError: Class constructor CompositeLayer cannot be invoked without 'new'

```
import { Map } from "maplibre-gl";
import { MapboxLayer } from "@deck.gl/mapbox";
import FlowMapLayer from "@flowmap.gl/core";

const map = new Map({
  container: "map",
  style: "https://basemaps.cartocdn.com/gl/positron-gl-style/style.json",
});

map.on("load", () => {
  map.addLayer(
    new MapboxLayer({
      id: "deckgl-flow",
      type: FlowMapLayer,
      locations: [
        {
          id: 1,
          name: "New York",
          lat: 40.713543,
          lon: -74.011219,
        },
        { id: 2, name: "London", lat: 51.507425, lon: -0.127738 },
        {
          id: 3,
          name: "Rio de Janeiro",
          lat: -22.906241,
          lon: -43.180244,
        },
      ],
      flows: [
        { origin: 1, dest: 2, count: 42 },
        { origin: 2, dest: 1, count: 51 },
        { origin: 3, dest: 1, count: 50 },
        { origin: 2, dest: 3, count: 40 },
        { origin: 1, dest: 3, count: 22 },
        { origin: 3, dest: 2, count: 42 },
      ],
      getFlowMagnitude: (flow) => flow.count || 0,
      getFlowOriginId: (flow) => flow.origin,
      getFlowDestId: (flow) => flow.dest,
      getLocationId: (loc) => loc.id,
      getLocationCentroid: (location) => [location.lon, location.lat],
    })
  );
});
```

<img width="1383" alt="Screenshot 2021-09-05 at 22 51 41" src="https://user-images.githubusercontent.com/548708/132141116-5a3f8eef-fff5-4742-ba47-4af3b2ae66cf.png">

https://github.com/MasterMaps/flowmap/blob/main/src/Map.js

Any idea what could be wrong?

**Comment by ilyabo** [comment]
Hm, I haven't tried it in the interleaved mode myself. Looks like it doesn't work well with CompositeLayers. Could you try to use [FlowLinesLayer](https://github.com/teralytics/flowmap.gl/blob/master/packages/core/src/FlowLinesLayer/FlowLinesLayer.ts) directly instead of FlowMapLayer?

**Comment by turban** [comment]
I also get an error with the FlowLinesLayer: 

<img width="1382" alt="Screenshot 2021-09-06 at 01 43 40" src="https://user-images.githubusercontent.com/548708/132144610-8da46b07-cc69-4f29-acee-9f5005d8f446.png">

Could it be a transpiler issue? https://stackoverflow.com/a/51860850

I see that ArcLayer is a class when I import it, while FlowMapLayer/FlowLinesLayer are functions. 

<img width="486" alt="Screenshot 2021-09-06 at 01 52 27" src="https://user-images.githubusercontent.com/548708/132144822-c2dc353e-bbb1-49b0-b330-e7d8bdeeaa87.png">
<img width="364" alt="Screenshot 2021-09-06 at 01 52 00" src="https://user-images.githubusercontent.com/548708/132144823-2fdf0325-27ff-46bb-a81f-68168da94573.png">

**Comment by neruyzo** [comment]
Hi,
I had the same problem with React Typescript environment, configure with create-react-app.
I manage to make a working @flowmap.gl/core package for my use (@meakio/core version:"^7.3.6" on npm for test).

I made change in tsconfig.build.esm according to the stackoverflow found by @turban.

```json
{
  "extends": "./tsconfig.build.json",
  "compilerOptions": {
    "target": "es2017",
    "module": "es2015",
    "outDir": "dist-esm",
    "moduleResolution": "node"
  }
}
```
I don't know if it broke other systems.

**Comment by ilyabo** [comment]
Ah, thanks @neruyzo! Could one of you guys create a PR?

**Comment by neruyzo** [comment]
@ilyabo here the PR https://github.com/teralytics/flowmap.gl/pull/125

**Comment by antoine-giret** [comment]
Hey ! Thanks for the fix. Is it possible to create a new release please?

**Comment by ilyabo** [comment]
Yes, I had some issues building Storybook after merging the changes. Will try to resolve and publish in the next few days.

**Comment by ilyabo** [comment]
Published 7.3.4, let me know if it solves the issue

**Comment by antoine-giret** [comment]
Yes everything is working now, thank you very much!
